src/styles/atoms/_atoms.table.scss

```diff
@@ -4,7 +4,7 @@
  */
 
 .a-table,
-.a-table__wrapper-responsive {
+.a-table-scrollable-wrapper {
```

Comment:
Moet het niet `.a-table__scrollable-wrapper` zijn?

Reply:
@TriangleJuice  We hadden vorige keer besproken om van die wrapper classes util-classes te maken: `a-table-scrollable-wrapper` => `u-table-scrollable-wrapper` en `a-table-responsive-wrapper` => `u-table-responsive-wrapper`. Er schiet mij nu te binnen dat die `a-table-responsive-wrapper` ook 4 modifier classes heeft. Dus `u-table-responsive-wrapper has-first-column-fix has-shadow-left has-last-column-fix has-shadow-right` is mogelijk. Util-classes die modifier classes hebben, lijkt me een beetje raar en komt nergens anders voor. Dus ik zou het eventueel toch `a-table-scrollable-wrapper` en `a-table-responsive-wrapper` houden. Of enkel `a-table-scrollable-wrapper` naar `u-table-scrollable-wrapper` veranderen. Wat denk je? 🙂

Reply:
Als de consistentie zo beter is, dan doe je het maar zo. Merci al voor de moeite! 😊